### PR TITLE
VZ-9206: Add new templates for calico

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

File: ociocne/capi/objects.go [hunk 1]
```diff
@@ -54,6 +54,9 @@ var ccm = []Object{
 var cni = []Object{
 	{GVR: gvr.ConfigMap, Text: templates.CalicoConfigMap},
```
Comment (anders-swanson) [inline ociocne/capi/objects.go:55]: missing other configmaps?

File: ociocne/templates/calicocustomresourcesconfigmap.goyaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      # Configures Calico networking.
+      calicoNetwork:
+        bgp: Disabled
+        # Note: The ipPools section cannot be modified post-install.
+        ipPools:
+        - cidr: {{ .PodCIDR }}
+          encapsulation: VXLAN
+          # natOutgoing: Enabled
+          # nodeSelector: all()
+      registry: {{ .ControlPlaneRegistry }}
```
Comment (anders-swanson) [inline ociocne/templates/calicocustomresourcesconfigmap.goyaml:22]: should be calico registry

File: ociocne/templates/calicocustomresourcesconfigmap.goyaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    kind: Installation
+    metadata:
+      name: default
+    spec:
+      # Configures Calico networking.
+      calicoNetwork:
+        bgp: Disabled
+        # Note: The ipPools section cannot be modified post-install.
+        ipPools:
+        - cidr: {{ .PodCIDR }}
```
Comment (anders-swanson) [inline ociocne/templates/calicocustomresourcesconfigmap.goyaml:18]: where does calico tag factor in?

File: ociocne/capi/objects.go (hunk 1 above)
Comment (theabmitra) [reply ociocne/capi/objects.go:55]: done

File: ociocne/templates/calicocustomresourcesconfigmap.goyaml (hunk 3 above)
Comment (theabmitra) [reply ociocne/templates/calicocustomresourcesconfigmap.goyaml:18]: Its tied to the tigera image version

File: ociocne/templates/calicocustomresourcesconfigmap.goyaml (hunk 2 above)
Comment (theabmitra) [reply ociocne/templates/calicocustomresourcesconfigmap.goyaml]: done

**Review by anders-swanson — APPROVED** [review]

**Pull request merged**